File: oop/ts-oop/Animal.ts
export class Animal {
    name : string ; 
    age : number ; 

    constructor(name : string  , age : number){
        this.name = name ;
        this.age = age;
    }

    echoName(){
        console.log(this.name);
    }

    getAge() : number {
        return this.age ;
    }
}

File: oop/javaoop/Animal.java
package oop.javaoop;

public class Animal {
    protected  String name  ; 
    protected  int age ; 

    public Animal(String name , int age){
        this.name = name ;
        this.age = age;
    }

    public void echoName(){
        System.out.println(this.name);
    }

    public int getAge(){
        return this.age ;
    }
}
